examples/recording_service/pluggable_storage/c++11/CMakeLists.txt

```diff
@@ -19,15 +19,17 @@ if(NOT BUILD_SHARED_LIBS)
     set(msg
         "Static linking is not supported for this example. You can build it "
         "using the -DBUILD_SHARED_LIBS=ON CMake parameter")
-    message(FATAL_ERROR ${msg})
+    message(WARNING ${msg})
+    return()
```

Comment:
Will the `return()` call stop the generate step? It wasn't super clear from the CMake documentation - https://cmake.org/cmake/help/latest/command/return.html